tests/robustness/engine/engine.go

```diff
@@ -231,6 +233,30 @@ func (e *Engine) CleanComponents() {
 	os.RemoveAll(e.baseDirPath) //nolint:errcheck
 }
 
+// CleanEngineComponents cleans up each component part of the test engine.
+func (e *Engine) CleanEngineComponents() {
+	for _, f := range e.cleanupRoutines["engine"] {
+		if f != nil {
+			f()
+		}
+	}
+
+	os.RemoveAll(e.baseDirPath) //nolint:errcheck
+}
+
+func (e *Engine) CleanAllComponents() {
+	log.Printf("Cleaning Whole")
```

Comment:
This method does not explicitly remove the baseDirPath; instead it would be removed by the first component cleanup method invoked.